src/view/track_view.py

```diff
@@ -9,49 +9,69 @@
 from pygame.surface import Surface
 
 from model.track.track import Track
-from model.simulation import Simulation
+from model.neural_network.neural_network import LayerInfo
+from model.neuroevolution.neuroevolution import Neuroevolution
 from view import colors
 from view.action import Action, ActionType
+from view.pygame_environment import PyGameEnvironment, EnvironmentContext
 from view.view import View
 
 
 class TrackView(View):
-    simulation: Optional[Simulation]
-    simulation_class: Type[Simulation]
-    dataset: Dict[str, Any]
-    board: Surface
-    coord_start: Vec2
+    generator: Generator[None, EnvironmentContext, None]
+    track: Track
+    environment: PyGameEnvironment
+
+    neuroevolution: Neuroevolution
+
+    _paused = False
+    input_neurons = 9
```

Comment:
@KamilKrol5, you said  something about 6 sensors and you use 8 here.

Reply:
There will be 6 sensors, as we decided.